### PR TITLE
v2.0.0 Alpha

### PR DESCRIPTION
**Pull request by douglasduteil**

### CODE REVIEW

**Comment by douglasduteil** [comment]
Closes #171

File: data/index.json [hunk 1]
```diff
@@ -3317,3 +3317,23 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (revolunet) [inline data/index.json:3339]: problem here ?

Comment (douglasduteil) [reply data/index.json:3339]: A merde le debug :sweat_smile: 
A fixer sur le master

Comment (douglasduteil) [reply data/index.json:3339]: Done with https://github.com/SocialGouv/legi-data/commit/431379de2bca2114a4406fc52343ce57fdb0cea0

**Pull request merged**